drivers/include/periph/flashpage.h

```diff
@@ -101,16 +127,39 @@ static inline int flashpage_page(void *addr)
  * @brief   Write the given page with the given data
  *
  * @param[in] page      page to write
- * @param[in] data      data to write to the page, MUST be @p FLASHPAGE_SIZE
+ * @param[in] data      data to write to the page, MUST be FLASHPAGE_SIZE
  *                      byte. Set to NULL for page erase only.
  */
 void flashpage_write(int page, void *data);
 
+/**
+ * @brief   Write any number of data bytes to a given location in the
+ *          flash memory
+ *
+ * @warning Make sure the targeted memory area is erased before calling
+ *          this function
+ *
+ * Both target address and data address must be aligned to
+ * FLASHPAGE_RAW_ALIGN. @p len must be a multiple of FLASHPAGE_RAW_BLOCKSIZE.
+ * This function doesn't erase any area in flash, thus be sure the block is
```

Comment:
Sorry @kYc0o, but I think this is the third time I raise this issue: writing about `the block` in this sentence is misleading! Better:
`This function does not not erase any parts of the flash, thus be sure that the targeted memory area has been erased before calling this function (using the flashpage_write() function).`

Reply:
It's very strange... can you check the last commit? Unless this phrase is in several places I could have missed it...

Reply:
Ohhh yeah it's twice! :/ my bad, will fix.

Reply:
Done.